tests/utils/tools.php

```diff
@@ -95,6 +95,25 @@ function get_primary_server($uri)
     return (new Manager($uri))->selectServer(new ReadPreference('primary'));
 }
 
+/**
+ * Runs a command and returns whether an exception was thrown or not
+ *
+ * @param string $uri Connection string
+ * @param Command $command
+ * @return bool
+ * @throws RuntimeException
+ */
+function command_works($uri, $command)
```

Comment:
Any reason not to type-hint `$command` here?

Alternatively, allow it to take any `array|object` and then wrap it in a Command as needed. This is what PHPLIB's [`DatabaseCommand`](https://github.com/mongodb/mongo-php-library/blob/1.4.1/src/Operation/DatabaseCommand.php#L81) helper does. It would save you from needing to import and construct `MongoDB\Driver\Command` from `skip_if_sleep_command_unavailable()`, and anything else that may use this in the future.